docs/reference/versions/index.md

```diff
@@ -1,35 +1,26 @@
 # Service version matrix
 
-Services in {{brand}} are updated on a regular basis and
-on a rolling schedule.
+Services in {{brand}} are updated on a regular basis and on a rolling schedule.
 
-This section lists the cloud API service versions available in each
-{{brand}} region.
+This section lists the cloud API service versions available in each {{brand}} region.
 
-* [Public Cloud](public.md): versions running in our {{company}}
-  Public Cloud regions.
+* [Public Cloud](public.md): versions running in our {{company}} Public Cloud regions.
 
-* [Compliant Cloud](compliant.md): versions running in our
-  {{company}} Compliant Cloud regions.
+* [Compliant Cloud](compliant.md): versions running in our {{company}} Compliant Cloud regions.
 
 
 ## OpenStack Services
 
-[OpenStack releases](https://releases.openstack.org) are named, in
-alphabetical order, and occur on a six-month release schedule. In
-{{company}} Public Cloud we upgrade OpenStack releases annually; this
-means that we deploy every other OpenStack release and skip the
-intervening one.
+[OpenStack releases](https://releases.openstack.org) are named in alphabetical order, and occur on a six-month release schedule.
+In {{company}} Public Cloud we upgrade OpenStack releases annually; this means that we normally deploy every other OpenStack release and skip the intervening one.
 
-{{brand}} currently runs OpenStack
-[Xena](https://releases.openstack.org/xena).
+{{brand}} currently runs OpenStack [Xena](https://releases.openstack.org/xena) and [Antelope](https://releases.openstack.org/antelope).
+The fact that {{brand}} has skipped the [Zed](https://releases.openstack.org/zed) release is due to a one-time upstream release policy change.
```

Comment:
We also skipped Yoga then. The reason why we skipped both is because we decided to align with SLURP schedule.

Reply:
That's what this says: in addition to the skipping the Yoga release (which is what we would would have done anyway, as the preceding paragraph explains), we also skipped Zed.

Reply:
@fghaas I see it as an improvement. As a reader, I appreciate that level of detail. If anything, it shows that the people behind the scenes have good reasons for whatever they do or don't do.

Reply:
@colder-is-better Fair. I've squashed that fixup, and updated the PR branch. Please re-approve so we can merge this.